git-sync/ssh-wrapper.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+secret_path=/etc/git-secret/ssh
```

Comment:
Not sure what you're asking:

1) Why we need to specify a path to the SSH key?
2) Why we need a variable secret_path? (we don't, could just put the path inline with the command)
3) Why we need the .sh file at all?

Also I suppose we could make the path configurable as an env var or something, but I thought it might just be easier to tell the user where to put it.